aries_cloudagent/wallet/models/tests/test_wallet_record.py

```diff
@@ -0,0 +1,66 @@
+from asynctest import TestCase as AsyncTestCase
+
+from .. import wallet_record as test_module
+from ..wallet_record import WalletRecord
+
+
+class TestIssuerCredRevRecord(AsyncTestCase):
```

Comment:
I think this needs renaming? It looks like it might duplicate some additional testing in PR #889 as well